components/esp_lcd_touch_ft5x06/esp_lcd_touch_ft5x06.c

```diff
@@ -290,32 +282,19 @@ static esp_err_t touch_ft5x06_init(esp_lcd_touch_handle_t tp)
 
 static esp_err_t touch_ft5x06_i2c_write(esp_lcd_touch_handle_t tp, uint8_t reg, uint8_t data)
 {
-    esp_lcd_touch_dev_i2c_t *ft5x06_dev = NULL;
-    uint8_t buff[] = {reg, data};
-
     assert(tp != NULL);
 
-    ft5x06_dev = &tp->config.device.i2c;
-    assert(ft5x06_dev->port >= 0 && ft5x06_dev->port < SOC_I2C_NUM);
-
-    /* Write register and data number */
-    return i2c_master_write_to_device(ft5x06_dev->port, ESP_LCD_TOUCH_FT5x06_I2C_ADDRESS, buff, sizeof(buff), ESP_LCD_TOUCH_FT5x06_I2C_TIMEOUT_MS / portTICK_PERIOD_MS);
+    // *INDENT-OFF*
+    /* Write data */
+    return esp_lcd_panel_io_tx_param(tp->io, reg, (uint8_t[]){data}, 1);
+    // *INDENT-ON*
```

Comment:
Nitpick: single line formating disable?

```suggestion
    /* Write data */
    return esp_lcd_panel_io_tx_param(tp->io, reg, (uint8_t[]){data}, 1);  // *INDENT-OFF*
```

Reply:
This is not working...